test/unit/alignment/matrix/debug_stream_alignment_coordinate_test.cpp

```diff
@@ -11,27 +11,28 @@
 
 #include <seqan3/alignment/matrix/alignment_coordinate.hpp>
 
-using namespace seqan3;
-
 TEST(debug_stream_test, advanceable_alignment_coordinate)
 {
     using not_incrementable =
-        detail::advanceable_alignment_coordinate<detail::advanceable_alignment_coordinate_state::none>;
+        seqan3::detail::advanceable_alignment_coordinate<seqan3::detail::advanceable_alignment_coordinate_state::none>;
     using row_incrementable =
-        detail::advanceable_alignment_coordinate<detail::advanceable_alignment_coordinate_state::row>;
+        seqan3::detail::advanceable_alignment_coordinate<seqan3::detail::advanceable_alignment_coordinate_state::row>;
     using col_incrementable =
-        detail::advanceable_alignment_coordinate<detail::advanceable_alignment_coordinate_state::column>;
+        seqan3::detail::advanceable_alignment_coordinate<seqan3::detail::advanceable_alignment_coordinate_state::column>;
 
-    not_incrementable co_not{detail::column_index_type{10u}, detail::row_index_type{5u}};
-    col_incrementable co_col{detail::column_index_type{10u}, detail::row_index_type{5u}};
-    row_incrementable co_row{detail::column_index_type{10u}, detail::row_index_type{5u}};
+    not_incrementable co_not{seqan3::detail::column_index_type{10u}, seqan3::detail::row_index_type{5u}};
+    col_incrementable co_col{seqan3::detail::column_index_type{10u}, seqan3::detail::row_index_type{5u}};
+    row_incrementable co_row{seqan3::detail::column_index_type{10u}, seqan3::detail::row_index_type{5u}};
 
-    EXPECT_TRUE((detail::is_value_specialisation_of_v<not_incrementable, detail::advanceable_alignment_coordinate>));
-    EXPECT_TRUE((detail::is_value_specialisation_of_v<col_incrementable, detail::advanceable_alignment_coordinate>));
-    EXPECT_TRUE((detail::is_value_specialisation_of_v<row_incrementable, detail::advanceable_alignment_coordinate>));
+    EXPECT_TRUE((seqan3::detail::is_value_specialisation_of_v<not_incrementable,
+                 seqan3::detail::advanceable_alignment_coordinate>));
```

Comment:
```suggestion
    EXPECT_TRUE((seqan3::detail::is_value_specialisation_of_v<not_incrementable,
                                                              seqan3::detail::advanceable_alignment_coordinate>));
```

Also for the other 3 occurrences in this file